sites/public/src/layouts/application.tsx

```diff
@@ -21,12 +20,83 @@ const Layout = (props) => {
       </div>
 
       <SiteFooter>
-        <FooterNav copyright={t("footer.copyright")}>
-          <Link href="/privacy">{t("pageTitle.privacy")}</Link>
-          <Link href="/disclaimer">{t("pageTitle.disclaimer")}</Link>
-        </FooterNav>
-        <FooterSection className="bg-black" small>
-          <ExygyFooter />
+        <FooterSection sectionClassName="justify-around" small>
+          <div className="text-left">
+            <img
+              className="h-20 w-20 mr-3"
+              src="/images/bahfa-logo.png"
+              alt="Bay Area Housing Finance Authority Logo"
+            />
+            <img
+              className="h-24 w-96"
+              src="/images/mtc-abag-logo.svg"
+              alt="Association of Bay Area Governments - Metropolitan Transporation Commission Logo"
+            />
+            <p className="mt-8 text-white">
+              375 Beale Street, Suite 800
+              <br /> San Francisco, CA 94105-2066
+              <br /> {t("footer.mondayToFriday")}
+              <br />
+              <a href="mailto: bahfa@bayareametro.gov" className="underline">
```

Comment:
can we use `Link` components instead of `a` tags?

Reply:
I had that at first but the next documentation mentions that this is beneficial for linking to internal pages https://nextjs.org/learn/basics/navigate-between-pages/link-component but since these are external sites, it doesn't seem like it's doing anything